src/widget/city_with_overlay.c

```diff
@@ -3,6 +3,7 @@
 #include "building/animation.h"
 #include "building/construction.h"
 #include "building/industry.h"
+#include "core/config.h"
 #include "city/view.h"
```

Comment:
Alphabetical order :)
```suggestion
#include "city/view.h"
#include "core/config.h"
```